projects/graph/graph.py

```diff
@@ -17,43 +17,82 @@ def add_vertex(self, vertex):
     def add_edge(self, v1, v2):
         if v1 not in self.vertices:
             self.add_vertex(v1)
-        
+
         if v2 not in self.vertices:
             self.add_vertex(v2)
-        
+
         self.vertices[v1].add(v2)
-        
+
         self.vertices[v2].add(v1)
 
     def bft(self, starting_vertex):
```

Comment:
Your BFT algorithm looks great Michael 👍 , could throw a couple of print statements in there and check out how you example graphs get traversed to solidify your understanding.